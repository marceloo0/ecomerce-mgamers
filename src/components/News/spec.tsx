import { screen } from '@testing-library/react'
import { renderWithTheme } from '../../utils/tests/helpers'

import { News } from '.'

describe('<News />', () => {
  it('should render default', () => {
    const { container } = renderWithTheme(<News />)

    expect(screen.getByLabelText(/news/i)).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
