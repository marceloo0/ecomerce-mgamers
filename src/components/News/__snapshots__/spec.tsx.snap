// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<News /> should render default 1`] = `
.c2 {
  height: 100%;
  width: 100%;
  z-index: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c3 {
  font-size: 3.8rem;
  font-weight: 600;
  color: #2E2F42;
}

.c0 {
  background-color: #201C2B;
  height: 100%;
  position: relative;
}

.c1 {
  position: absolute;
  left: auto;
  top: 4rem;
  z-index: 0;
}

.c5 {
  width: 100%;
  display: grid;
  grid-template-rows: 1fr;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

@media screen and (min-width:768px) and (max-width:1023px) {
  .c3 {
    font-size: 7rem;
  }
}

@media (min-width:1024px) {
  .c3 {
    font-size: 10rem;
  }
}

@media screen and (min-width:768px) {
  .c5 {
    width: 100%;
    height: 100%;
    margin-top: 8rem;
    display: grid;
    grid-template-columns: repeat(2,1fr);
  }
}

@media screen and (min-width:1024px) {
  .c5 {
    width: 100%;
    height: 100%;
    margin-top: 8rem;
    display: grid;
    grid-template-columns: repeat(5,1fr);
  }
}

<div
  aria-label="news"
  class="c0"
>
  <div
    class="c1"
  >
    <div
      class="c2"
    >
      <span
        class="c3"
      >
        NovidadeS
      </span>
    </div>
  </div>
  <div
    class="c4"
  >
    <div
      class="c5"
    />
  </div>
</div>
`;
